ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
OK, I'm also fine with this if you @b49020 :
a) Add the `--no-builtin-variables` as I suggested in my comment above.
b) Add the clarifying comment proposed by Jerome just above.

Please also rebase and add our tags and I'll merge that patch.

`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`